android-connection/noto-fonts-4-android.xml

```diff
@@ -1804,4 +1844,27 @@
             NotoSerifOldUyghur-Regular.ttf
         </font>
 	</family>
+	<family lang="und-Cpmn">
```

Comment:
description might need an update to account for these changes? - it says just Oriya weights